src/renderers/WebGLRenderer.js

```diff
@@ -3030,10 +3033,12 @@ THREE.WebGLRenderer = function ( parameters ) {
 
 	this.allocTextureUnit = allocTextureUnit;
 
+	//this.setTexture2D = setTexture2D;
```

Comment:
No! Please keep it!

Why? These routines are actually interchangeable minus deprecation, the others are not.